docs/data/date-pickers/fields/LifeCycleIgnoreInvalidValue.js

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
+
+export default function LifeCycleIgnoreInvalidValue() {
+  const [value, setValue] = React.useState(null);
+
+  return (
+    <Stack spacing={2}>
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
+        <DateField
+          value={value}
+          onChange={(newValue, context) => {
+            if (context.validationError == null) {
+              setValue(newValue);
+            }
+          }}
+          minDate={dayjs('2022-04-01')}
+          maxDate={dayjs('2022-04-30')}
```

Comment:
April 2022 is maybe too restrictive for a fast reader. A full year might be easier to find (I know it's described but people tends to play with the demo before finishing the text)
```suggestion
          minDate={dayjs('2022-01-01')}
          maxDate={dayjs('2022-12-31')}
```

Reply:
You are probably right, updated :+1: